tests/Unit/Elliptic/DiscontinuousGalerkin/Test_ImposeBoundaryConditions.cpp

```diff
@@ -0,0 +1,487 @@
+  //        +-------+-+-+---+
+  // We run the actions on the indicated element.
+  const ElementId<2> self_id(1, {{{2, 0}, {1, 0}}});
+  const ElementId<2> west_id(0);
+  const ElementId<2> east_id(1, {{{2, 1}, {1, 0}}});
+  const ElementId<2> south_id(1, {{{2, 0}, {1, 1}}});
+
+  const CoordinateMaps::Affine xi_map{-1., 1., 3., 7.};
+  const CoordinateMaps::Affine eta_map{-1., 1., 7., 3.};
+
+  const auto coordmap =
+      make_coordinate_map_base<Frame::Logical, Frame::Inertial>(
+          CoordinateMaps::ProductOf2Maps<CoordinateMaps::Affine,
+                                         CoordinateMaps::Affine>(xi_map,
+                                                                 eta_map));
+
+  const auto external_directions = {Direction<2>::lower_eta(),
+                                    Direction<2>::upper_xi()};
+
+  const auto external_mortar_ids = {
+      std::make_pair(Direction<2>::lower_eta(),
+                     ElementId<2>::external_boundary_id()),
+      std::make_pair(Direction<2>::upper_xi(),
+                     ElementId<2>::external_boundary_id())};
+
+  const struct {
+    std::unordered_map<Direction<2>, Scalar<DataVector>> bdry_vars;
+    std::unordered_map<Direction<2>, Scalar<DataVector>> bdry_fluxes;
+    std::unordered_map<Direction<2>, Scalar<DataVector>> bdry_other_data;
+    std::unordered_map<Direction<2>, Scalar<DataVector>> exterior_bdry_fluxes;
+    std::unordered_map<Direction<2>, Scalar<DataVector>>
+        exterior_bdry_other_data;
+    std::unordered_map<Direction<2>, Scalar<DataVector>> exterior_bdry_vars;
+  } data{{{Direction<2>::lower_eta(), Scalar<DataVector>{{{{-1., -2., -3.}}}}},
+          {Direction<2>::upper_xi(), Scalar<DataVector>{{{{-4., -5., -6.}}}}}},
+         {{Direction<2>::lower_eta(), Scalar<DataVector>{{{{1., 2., 3.}}}}},
+          {Direction<2>::upper_xi(), Scalar<DataVector>{{{{21., 22., 23.}}}}}},
+         {{Direction<2>::lower_eta(), Scalar<DataVector>{{{{4., 5., 6.}}}}},
+          {Direction<2>::upper_xi(), Scalar<DataVector>{{{{24., 25., 26.}}}}}},
+         {{Direction<2>::lower_eta(), Scalar<DataVector>{{{{7., 8., 9.}}}}},
+          {Direction<2>::upper_xi(), Scalar<DataVector>{{{{27., 28., 29.}}}}}},
+         {{Direction<2>::lower_eta(), Scalar<DataVector>{{{{10., 11., 12.}}}}},
+          {Direction<2>::upper_xi(), Scalar<DataVector>{{{{30., 31., 32.}}}}}},
+         {{Direction<2>::lower_eta(), Scalar<DataVector>{{{{13., 14., 15.}}}}},
+          {Direction<2>::upper_xi(), Scalar<DataVector>{{{{33., 34., 35.}}}}}}};
+
+  auto start_box = [
+    &mesh, &self_id, &west_id, &south_id, &coordmap, &data,
+    &external_directions, &external_mortar_ids
+  ]() noexcept {
+    const Element<2> element(self_id,
+                             {{Direction<2>::lower_xi(), {{west_id}, {}}},
+                              {Direction<2>::upper_eta(), {{south_id}, {}}}});
+
+    auto map = ElementMap<2, Frame::Inertial>(self_id, coordmap->get_clone());
```

Comment:
I think it is a good recommendation to not use names which fit entirely within other names (for example `map` will fit into `coord_map`). It is a good practice to use names not contained in other names so that we can both:
1 - know what distinguishes this variable from the other variable
2 - have an easier time using find and replace if it ever becomes necessary :octocat: